pyracing/client.py

```diff
@@ -114,9 +134,18 @@ async def active_op_counts(
             'excludeLite': None  # Purpose of excludeLite unknown
         }
         response = await self._build_request(url, payload)
-
-        return [upcoming_events.OpenPractice(x) for x in response.json()["d"]]
-
+        
+        if response.json():
+            mapping = response.json().get('m')
+            response_items = response.json().get('d')
+        
+            if mapping and response_items:
+                renamed_items = [self._rename_numerical_keys(ri, mapping) for ri in response_items]
+                
+                return [upcoming_events.OpenPractice(x) for x in renamed_items]
```

Comment:
Rename numerical keys can return a `None`, but this is trying to iterate over it. If we get a `None` response from `_rename_numerical_keys` then it will try to create an `OpenPractice` object from `None`, which will fail. We should either remove the `None`s from the list, or find something else to return such that this is safer

Reply:
Sorry, I know I am really hammering this one, but I think once this is fixed I am ready to approve this. Just want to make sure if the API does fail, that we return an empty list or less data, and not crash entirely

Reply:
No problem, I changed the lines so they check if x exists, before creating the response_objects: 
`return [upcoming_events.OpenPractice(x) for x in renamed_items if x]` for example...